iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverterTest.java

```diff
@@ -0,0 +1,168 @@
+import eu.dnetlib.data.proto.ProjectOrganizationProtos.ProjectOrganization.Participation;
+import eu.dnetlib.data.proto.RelMetadataProtos.RelMetadata;
+import eu.dnetlib.data.proto.RelTypeProtos.RelType;
+import eu.dnetlib.data.proto.RelTypeProtos.SubRelType;
+import eu.dnetlib.iis.importer.schemas.ProjectToOrganization;
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+* @author mhorst
+*/
+
+public class ProjectToOrganizationConverterTest {
+
+	private byte[] relationColumnFamilyBytes = "dummy".getBytes();
+	
+    private Oaf oaf = null;
+    
+    private Result result = mock(Result.class);
+    
+    private ResultApprover defaultApprover = new ResultApprover() {
+		@Override
+		public boolean approveBeforeBuilding(Oaf oaf) {
+			return true;
+		}
+	};
+    
+    
+    //------------------------ TESTS --------------------------
+    
+	@Test(expected = NullPointerException.class)
+    public void constructor_relationColumnFamilyBytes_NULL() throws Exception {
+    	new ProjectToOrganizationConverter(defaultApprover, null);
+    }
+	
+	@Test(expected = NullPointerException.class)
+    public void constructor_resultApprover_NULL() throws Exception {
+    	new ProjectToOrganizationConverter(null, relationColumnFamilyBytes);
+    }
+	
+    @Test(expected = NullPointerException.class)
+    public void buildObject_hbaseResult_NULL() throws Exception {
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			defaultApprover, relationColumnFamilyBytes);
+        converter.buildObject(null, oaf);
+    }
+    
+    @Test
+    public void buildObject_disapproved_candidates() throws Exception {
+        //given
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			new ResultApprover() {
+					@Override
+					public boolean approveBeforeBuilding(Oaf oaf) {
+						return false;
+					}
+				}, relationColumnFamilyBytes);
+    	
+    	String projectId = "someProjectId";
+    	String organizationId = "someOrgId";
+    	
+    	NavigableMap<byte[],byte[]> navigableMap = new TreeMap<byte[],byte[]>(
+    			UnsignedBytes.lexicographicalComparator());
+    	navigableMap.put(organizationId.getBytes(), createOafObject(projectId, organizationId).toByteArray());
+    	when(result.getFamilyMap(relationColumnFamilyBytes)).thenReturn(navigableMap);
+
+        // execute 
+        Collection<ProjectToOrganization> projOrgs = converter.buildObject(result, oaf);
+        
+        // assert
+        assertNotNull(projOrgs);
+        assertTrue(projOrgs.isEmpty());
+    }
+    
+    @Test
+    public void buildObject_no_candidates() throws Exception {
+        //given
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			defaultApprover, relationColumnFamilyBytes);
+    	
+    	when(result.getFamilyMap(relationColumnFamilyBytes)).thenReturn(new TreeMap<byte[],byte[]>(
```

Comment:
Is there a reason why you break the lines so early? The code is harder to read if the breaks are frequent.